### ISSUE TITLE
Android - General - TalkBack Needs More Informatin about Bookmark for User

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#15]
When using TalkBack, it reads the “bookmark button” without telling the user what it is bookmarking. I would expect TalkBack to read either “Bookmark SCREEN TITLE” or “SCREEN TITLE Bookmark” that way the user knows what they are bookmarking/saving.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

We've updated this and will be demo'ed on the next 508 compliance check after Sprint 7.

Best,
Kyle

**Comment by KyleLivingstonDOL** [comment]
It looks like this is done. UAT for Sprint 7 (which is ongoing right now) will start on June 10.

**Comment by jfc3-dol** [comment]
TalkBack is now telling the user to double tap to activate but not what is being bookmarked. 

I would expect TalkBack to read either “Bookmark SCREEN TITLE” or “SCREEN TITLE Bookmark” that way the user knows what they are bookmarking/saving.

**Comment by KyleLivingstonDOL** [comment]
ZTI, can you please address?

**Comment by jfc3-dol** [comment]
TalkBack is still not telling the screen it is bookmarking when you get to the bookmark button at the top of the page.

**Comment by KyleLivingstonDOL** [comment]
This should be fixed now

**Comment by jfc3-dol** [comment]
I will test is when the next build is pushed. Any idea when that will be?

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL 
Nice improvement, I like how the bookmark now tells the user what page/screen it's bookmarking, and it's a button. But for some reason, the text "Comply Chain United States Department of Labor" is in the information the button announces. That is a bit too much information for the user. By telling the user the screen, they are bookmarking and that it's a button that can be double-tapped is enough.

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL,
For some reason, the text "Comply Chain United States Department of Labor" is in the information the button announces. That is a bit too much information for the user. By telling the user the screen, they are bookmarking and that it's a button that can be double-tapped is enough.
